src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/dto/prisoner/search/PrisonerSearchResultDto.kt

```diff
@@ -11,6 +11,6 @@ data class PrisonerSearchResultDto(
   val lastName: String,
 ) {
   override fun toString(): String {
-    return "$firstName $lastName"
+    return "Your visit to see $firstName $lastName"
```

Comment:
sorry I know I suggested this but given that we are now appending "Your visit to see" could we move it into a different method in the service?